src/pymd4c.c

```diff
@@ -420,37 +420,50 @@ static int GenericParser_block(MD_BLOCKTYPE type, void *detail,
     PyObject *arglist;
     switch(type) {
         case MD_BLOCK_UL:
-            arglist = Py_BuildValue("(O{s:i,s:C})", get_enum_blocktype(type),
-                    "is_tight", ((MD_BLOCK_UL_DETAIL *) detail)->is_tight,
+            arglist = Py_BuildValue("(O{s:O,s:C})", get_enum_blocktype(type),
```

Comment:
`PyBool_FromLong()` creates a new reference. `O` should be `N` to avoid leaking it.

```suggestion
            arglist = Py_BuildValue("(O{s:N,s:C})", get_enum_blocktype(type),
```